services/src/contexts/migrations/database_migration.rs

```diff
@@ -106,24 +108,24 @@ where
             }
         );
 
-        info!(target: prefix, "Applying migration: {}", migration.version());
+        info!("Applying migration: {}", migration.version());
```

Comment:
if the database schema is created from scratch it should not speak of migrations but say "initializing database" IMHO

Reply:
[eaf44d5](https://github.com/geo-engine/geoengine/pull/945/commits/eaf44d5dbb00383dc5c00149813d0f445adcb75e)